app/services/senders/card_sender.rb

```diff
@@ -0,0 +1,80 @@
+class CardSender < AbstractSender
+
+  # How to use:
+  # sender = CardSender.new(recipient)
+  # sender.add_element(title: 'Invite a friend!', image_url: 'http...')
+  #         .add_url_button(title: 'Invite!', url: 'http...', webview_size: 'tall')
+  #         .add_share_button
+  # sender.add_element(...)
+  # sender.deliver!
+
+  def initialize(recipient)
+    super(recipient)
+    @elements = []
+  end
+
+  def add_element(title: nil, subtitle: nil, item_url: nil, image_url: nil)
```

Comment:
Hound is sortof right on the first one, but, this just the most straightforward way to do this (other than creating a button class).

Up'ed the line length for methods to 50.

:grimacing: